azure-pipelines.yml

```diff
@@ -35,10 +45,13 @@ stages:
       inputs:
         source: specific
         project: build
-        pipeline: 1
+        pipeline: 142
         artifact: sonic-buildimage.vs
         runVersion: 'latestFromBranch'
-        runBranch: 'refs/heads/master'
+        runBranch: 'refs/heads/$(BUILD_BRANCH)'
+        patterns: |
+            target/debs/bullseye/libyang*.deb
```

Comment:
Can you define a variable for debian release to replace a fixed one - 'bullseye'?

Reply:
I prefer to keep hardcoded deb release names. Usually, deb release upgrades will happen in steps in the master branch, and we may end up in a situation where libyang.deb and sonic-yang-models.whl are built with different toolchains. Also, it helps community developers to grep for such usages when he/she is upgrading the debian toolchain.